src/backend/utils/storage.js

```diff
@@ -34,6 +34,8 @@ module.exports = {
       .hmset(
         // using guid as keys as it is unique to posts
         post.guid,
+        'guid',
+        post.guid,
```

Comment:
#427 just landed and it made these changes. This is no longer needed

Reply:
Good catch.  I've rebased and updated this, new commits pushed to this branch.